Workflows/GIT/Branches.md

```diff
@@ -0,0 +1,33 @@
+Our Git process is an upgraded version of [Github Flow](https://guides.github.com/introduction/flow/).
+
+![](https://ftp.infinum.co/stjepan_hadjic/git-flow-2.jpg)
+
+Instead of a single `master` branch, we use two branches to record the history of the project. The `master` branch stores code that's currently in production. The `staging` branch serves as an integration branch for features and fixes. It holds data that reflects what's currently on staging.
+
+**The master branch is always deployable.**
+
+As we use [Productive](https://productive.io) to manage our tasks, we use **feature branches** for fixes and new features. Our branch naming convention is {type}/{task-number}{descriptive-task-name}. For example, a branch for adding authentication would be named `feature/123-authentication`.
+
+*Types*
+* _feature_ - new feature or an improvement to an existing feature
+* _fix_ - non-critical bugfix, improvement, paying the technical debt. Goes through code review process
+* _hotfix_ - time sensitive critical bugfix, that should be deployed to production as soon as possible. Not necessary that it goes through code review, but it should be revisited at a later stage, and properly fixed or improved
```

Comment:
<img width="822" alt="Branches_-_Rails_Handbook" src="https://user-images.githubusercontent.com/6377753/85986341-5f450280-b9ec-11ea-9959-03b03169f774.png">

you need to format this a bit :D